set.mm

```diff
@@ -522969,6 +523036,135 @@ Real and complex numbers (cont.)
       df-pred ) BCDBCEAFZGHIZJKZABLBCSMZJKZABLABCNUCUAABUBTJBCSROPQ $.
   $}
 
+
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d B z $.  $d ph x $.
+    $d x y $.  $d x z $.  $d y z $.
+    $( A version of ~ ralxp with explicit substitution.  (Contributed by Scott
+       Fenton, 21-Aug-2024.) $)
+    ralxpes $p |- ( A. x e. ( A X. B ) [. ( 1st ` x ) / y ].
+     [. ( 2nd ` x ) / z ]. ph <-> A. y e. A A. z e. B ph ) $=
+      ( cv c2nd cfv wsbc c1st nfsbc1v nfcv nfsbcw nfv sbcopeq1a ralxpf ) ADBGZH
+      IZJZCRKIZJABCDEFTCUALTDCUADUAMADSLNABOACDRPQ $.
+  $}
+
+  $( Ordered triple membership in a triple cross product.  (Contributed by
+     Scott Fenton, 21-Aug-2024.) $)
+  ot2elxp $p |- ( <. <. A , B >. , C >. e. ( ( D X. E ) X. F ) <->
+  ( A e. D /\ B e. E /\ C e. F ) ) $=
+    ( cop cxp wcel wa w3a opelxp anbi1i df-3an 3bitr4i ) ABGZDEHZIZCFIZJADIZBEI
+    ZJZSJPCGQFHITUASKRUBSABDELMPCQFLTUASNO $.
```

Comment:
Why not use the `<. A , B , C >.` notation for ordered triples? ([df-ot](https://us.metamath.org/mpeuni/df-ot.html))
Granted, it's equivalent.

Reply:
Mostly because I forgot about ordered triples. I'm thinking of revisiting these with OTs when I get around to it.